.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+// For more details, see https://aka.ms/devcontainer.json.
+{
+  "name": "Apple Pay",
+  "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}/advanced-integration/",
+  "image": "mcr.microsoft.com/devcontainers/javascript-node:20",
+  "postCreateCommand": "npm install",
+  "postAttachCommand": {
+    "Start server": "npx kill-port 8080 && npm start"
+  },
+  "forwardPorts": [
+    8080
+  ],
+  "portsAttributes": {
+    "8080": {
+      "label": "Preview of Apple Pay Flow",
+      "onAutoForward": "openPreview"
+    }
+  },
+  "secrets": {
+    "PAYPAL_CLIENT_ID": {
+      "description": "Sandbox client ID of the application.",
+      "documentationUrl": "https://developer.paypal.com/api/rest/"
+    },
+    "PAYPAL_APP_SECRET": {
+      "description": "Sandbox secret of the application.",
+      "documentationUrl": "https://developer.paypal.com/api/rest/"
+    }
```

Comment:
will need `MERCHANT_ID` as well.

Reply:
Merchant_Id is not available in .env.example file and so we have not included that in secrets. Let us know if this has to be part of secrets in devconatiner.json file.

Reply:
`Merchant_ID` is also a required field for Partner use cases.

  If its a direct merchant its not really Needed , But lets say the Integration is by Shopify for their Merchant 

ClientID will be Partner aka facilitator shopify in this case  and Merchant_ID is the Sub Merchant 
and  when Applicable `Merchant_ID` needs to be added as used here 
https://github.com/paypal-examples/applepay/blob/main/advanced-integration/views/checkout.ejs#L35

So if we want to keep same integration example of Individual Merchants and Partners is better to add it to the Docs

Reply:
@RatnadeepSimhadri Sure, will add the MERCHANT_ID to the secrets.
@KalpanaReddyC Could you please add the merchant_id